src/js/components/pages/greendash/MapCard.jsx

```diff
@@ -160,24 +163,36 @@ const SVGMap = ({ mapDefs, data, setFocusRegion, svgRef, showLabels, per1000 })
 			</path>
 		);
 
-		// Place label <text> elements. Skip regions with less than 100g carbon output. Harsh, I know
+		// Store temp label <text> elements. Skip regions with less than 100g carbon output. Harsh, I know
 		if (showLabels && carbon > 0.1 && pathCentres[pathId]) {
 			let { cx, cy } = { ...pathCentres[pathId], ...props };
 			const transY = mapDefs.svgAttributes.fontSize / 2;
 
-			labels.push(
-				<g key={`label-${id}`}>
-					<text className="map-label-name" x={cx} y={cy} textAnchor="middle" transform={`translate(0 ${-transY})`} fontWeight="600">
-						{props.name}
-					</text>
-					<text className="map-label-carbon" x={cx} y={cy} textAnchor="middle" transform={`translate(0 ${transY})`}>
-						{carbon.toFixed(2)} {unit}
-					</text>
-				</g>
-			);
+			tempMapLables.push({
+				carbon:carbon, 
+				label:(
+					<g key={`label-${id}`}>
+						<text className="map-label-name" x={cx} y={cy} textAnchor="middle" transform={`translate(0 ${-transY})`} fontWeight="600" style={{pointerEvents:"none"}}>
+							{props.name}
+						</text>
+						<text className="map-label-carbon" x={cx} y={cy} textAnchor="middle" transform={`translate(0 ${transY})`} style={{pointerEvents:"none"}}>
+							{carbon.toFixed(2)} {unit}
+						</text>
+					</g>
+				)}
+			)
 		}
 	});
 
+	if(showLabels){
+		// for each label we stored, only place the label if that region is responsible for more than 1% of emissions
+		tempMapLables.forEach((region) => {
+			if(region.carbon > totalCarbon / divisor / 100){
```

Comment:
Hello @LewisFramps 
For this line -- please use brackets to make the a / b / c clear. I wouldn't know without testing whether this does what you want or not.
Thanks.